### PR TITLE
releases,toc: add v3.1.0-beta.2 Release Notes

### PR DESCRIPTION
**Pull request by junlan-zhang**
### What is changed, added or deleted? 
- Add TiDB v3.1.0-beta.2 Release Notes

### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with:
- [x] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

**Comment by juliezhang1112** [comment]
PTAL @lilin90 Thanks!

File: releases/3.1.0-beta.2.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+TiDB 版本：3.1.0-beta.2
+
+TiDB Ansible 版本：3.1.0-beta.2
+
+## TiDB
+
+- 支持在列属性上添加 `AutoRandom` 关键字，控制系统自动为主键分配随机整数，避免 `AUTO_INCREMENT` 自增主键带来的写入热点问题 [#14555](http://github.com/pingcap/tidb/pull/14555)
+- 新增通过 DDL 语句为表创建、删除列存储副本的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
+- 新增优化器可自主选择不同的存储引擎的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
+- 新增 SQL Hint 支持不同的存储存储引擎的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
```
Comment (ran-huang) [inline releases/3.1.0-beta.2.md]: Is this a typo? @TomShawn 
```suggestion
- 新增 SQL Hint 支持不同的存储引擎的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
```

Comment (TomShawn) [reply releases/3.1.0-beta.2.md]: Thanks! Typo fixed.

File: releases/3.1.0-beta.2.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- 修复因 rule checker 给 Region 分配 store 失败导致系统 panic 的问题 [#2161](https://github.com/pingcap/pd/pull/2161)
+
+## TiFlash
+
+- 初始版本
+
+## Tools
+
++ TiDB Lightning
+    - 优化配置项，部分配置项在没有进行配置的时候使用[文档](/reference/tools/tidb-lightning/config.md)中的默认配置 [#255](https://github.com/pingcap/tidb-lightning/pull/255)
```
Comment (ran-huang) [inline releases/3.1.0-beta.2.md]: Is it better to use a clear link text that describes the content of this document?
```suggestion
    - 优化配置项，部分配置项在没有进行配置的时候使用[TiDB Lightning 配置参数](/reference/tools/tidb-lightning/config.md)中的默认配置 [#255](https://github.com/pingcap/tidb-lightning/pull/255)
```

Comment (TomShawn) [reply releases/3.1.0-beta.2.md]: Yes, it is.

File: releases/3.1.0-beta.2.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - 修复在非 Server mode 模式下 web 界面无法打开的问题 [#259](https://github.com/pingcap/tidb-lightning/pull/259)
++ BR
+    - 修复在恢复数据过程中遇到不可恢复的错误时，程序无法及时退出的问题 [#152](https://github.com/pingcap/br/pull/152)
+
+## TiDB Ansible
+
+- 修复在某些场景下获取不到 PD Leader 导致滚动升级命令执行失败的问题 [#1122](https://github.com/pingcap/tidb-ansible/pull/1122)
+- 新增同一个集群中部署多个 Grafana/Prometheus/Alertmanager 的功能 [#1143](https://github.com/pingcap/tidb-ansible/pull/1143)
+- 新增部署 TiFlash 组件的功能 [#1148](https://github.com/pingcap/tidb-ansible/pull/1148)
+- 新增 TiFlash 组件相关的的监控指标 [#1152](https://github.com/pingcap/tidb-ansible/pull/1152)
```
Comment (ran-huang) [inline releases/3.1.0-beta.2.md]:
```suggestion
- 新增 TiFlash 组件相关的监控指标 [#1152](https://github.com/pingcap/tidb-ansible/pull/1152)
```

**Pull request merged**